### ISSUE TITLE
[BUG] Investigate and Solve Data Tracker Contractor Work Orders Unexpected Contractor Name Change 

### ISSUE THREAD

**Issue by atdservicebot** [cityofaustin/atd-data-tech#17706]
> What application are you using?

AMD Data Tracker

> Describe the problem.

To whom it may concern:
I'm reporting a bug where work orders I created have been modified due to the actions of others, where that individual did not modify the field in question. The lead technician field in the screenshots I'm attaching was changed in a number of instances from contractor 'Mastec' to the contractor 'EARTC' in these instances after the work order inventory status that is part of the work order was modified by Ronnie Bell from 'JV Status' pending to 'complete' in a separate splash screen (finance functions), where there is no option to modify the 'lead technician' value. I have spoken with Ronnie and he assures me he did not modify the 'Lead Technician' field and demonstrated his editing process via Teams, despite the work order featuring a 'last modified' date with his name. The attachments show the fields in question highlighted (see file attachments)

> Is there anything else we should know?

see attached images

> Website Address

https://atd.knack.com/amd#work-orders/work-order-details/661ec8f703dfaa0028a88cf0/

Internet Browser: Chrome

> Requested By
Markus R.

[Attachment](https://api.knack.com/v1/applications/595d00ebd315cc4cb98daff4/download/asset/66620a7e7576fc0027845656/wo_contractor_name_changed3.png) (161.95kb)

Request ID: DTS24-115370

**Comment by KaroEngstrom** [comment]
## Thu 6/6/24
- I did some preliminary troubleshooting for the attached WO and found it be, from the start, the same contractor it is now, never changed and originally created by the submitter ([changelog](https://builder.knack.com/atd/amd/records/objects/object_31/record/661ec8f703dfaa0028a88cf0/history?builder_table_search=230865))

**Comment by ChristinaTremel** [comment]
- Synced with Markus and showed him the change log records showing that EARTC was the contractor on the work order since its creation. We filtered down the records and found that 9 of them have EARTC as the contractor instead of Mastec. These records also show that the contractor has been EARTC since the creation. 
- We went ahead and updated the lead technician to Mastec for these 9 work orders
![image](https://github.com/cityofaustin/atd-data-tech/assets/61209591/8f8370af-e2ef-4650-a8ed-dcf8d3943557)
![image](https://github.com/cityofaustin/atd-data-tech/assets/61209591/9b048c17-50ed-445f-9ed1-787778ebd8f5)
- Markus was super appreciative of the help with doing this in a batch way!
